### PR TITLE
test: parallel database tests

### PR DESCRIPTION
**Pull request by victoraugustolls**
This PR changes the database tests to run in parallel. E2E tests will be changed to use the same setup in another PR.

### CODE REVIEW

File: app/gateways/db/postgres/ledger/create_transaction_test.go [hunk 1]
```diff
@@ -210,10 +217,10 @@ func TestLedgerRepository_CreateTransactionFailure(t *testing.T) {
 
 	for _, tt := range testCases {
```
Comment (thalissonfelipe) [inline app/gateways/db/postgres/ledger/create_transaction_test.go:218]: Is the `tt := tt` statement missing here?

Comment (victoraugustolls) [reply app/gateways/db/postgres/ledger/create_transaction_test.go:218]: Yes! Missed this one, thanks!

File: app/gateways/db/postgres/ledger/create_transaction_test.go [hunk 2]
```diff
@@ -143,14 +150,14 @@ func TestLedgerRepository_CreateTransactionFailure(t *testing.T) {
```
Comment (thalissonfelipe) [inline app/gateways/db/postgres/ledger/create_transaction_test.go:150]: Is the `t.Parallel()`statement at the top of the function missing?

File: app/gateways/db/postgres/ledger/create_transaction_test.go (hunk 1 above)
Comment (victoraugustolls) [reply app/gateways/db/postgres/ledger/create_transaction_test.go:220]: Fixed! This was not parallel before.

File: app/gateways/db/postgres/ledger/create_transaction_test.go (hunk 2 above)
Comment (victoraugustolls) [reply app/gateways/db/postgres/ledger/create_transaction_test.go:152]: Fixed when looking at yours previous comment!

File: app/gateways/db/postgres/migrations/migrations.go [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+	"github.com/golang-migrate/migrate/v4/source/iofs"
+)
+
+//go:embed *.sql
+var migrations embed.FS
+
+func GetMigrationHandler(dbUrl string) (*migrate.Migrate, error) {
+	source, err := iofs.New(migrations, ".")
+	if err != nil {
+		return nil, fmt.Errorf("init httpfs: %w", err)
```
Comment (thalissonfelipe) [inline app/gateways/db/postgres/migrations/migrations.go:19]: Maybe update error message for "init **iofs**: %w"?

Comment (victoraugustolls) [reply app/gateways/db/postgres/migrations/migrations.go]: done!

**Review by thalissonfelipe — DISMISSED** [review]

File: app/gateways/db/postgres/ledger/get_synthetic_report_test.go [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+	"github.com/stretchr/testify/require"
+
+	"github.com/stone-co/the-amazing-ledger/app/domain/instrumentators"
+	"github.com/stone-co/the-amazing-ledger/app/domain/vos"
+)
+
+func TestLedgerRepository_GetSyntheticReportSuccess(t *testing.T) {
+	t.Parallel()
+
+	base := "liability.assets"
```
Comment (helder-jaspion) [inline app/gateways/db/postgres/ledger/get_synthetic_report_test.go:18]:
```suggestion
	const base = "liability.assets"
```

**Review by Streppel — DISMISSED** [review]

**Review by helder-jaspion — APPROVED** [review]

**Review by helder-jaspion — APPROVED** [review]

**Review by Streppel — APPROVED** [review]

**Pull request merged**